migration.json

```diff
@@ -726,6 +855,25 @@
         "OrganizationRegister": "0x2C27710BbBad69793Ce5EAe77093acb28fbe63c8",
         "Redeemer": "0x236e077F979f4CCB5F9a9e0163728FE2E743a56d",
         "GenericScheme": "0x199719EE4d5DCF174B80b80afa1FE4a8e5b0E3A0"
+      },
+      "0.0.1-rc.27": {
+        "GEN": "0xe78A0F7E598Cc8b0Bb87894B0F60dD2a88d6a8Ab",
```

Comment:
Yeah, I had to do this manually (because we didn't merge the previous package release) and apparently copied ganache addresses by mistake, fixed that.